src/main/kotlin/com/github/devcordde/devcordbot/commands/moderation/BlacklistCommand.kt

```diff
@@ -25,44 +25,52 @@ import com.github.devcordde.devcordbot.command.permission.Permission
 import com.github.devcordde.devcordbot.constants.Embeds
 import com.github.devcordde.devcordbot.database.DatabaseDevCordUser
 import com.github.devcordde.devcordbot.database.Users
+import net.dv8tion.jda.api.requests.restaction.CommandUpdateAction
 import org.jetbrains.exposed.sql.transactions.transaction
 
 /**
  * BlacklistCommand
  */
-class BlacklistCommand : AbstractCommand() {
-    override val aliases: List<String> = listOf("blacklist", "bl")
-    override val displayName: String = "blacklist"
+class BlacklistCommand : AbstractRootCommand() {
+    override val name: String = "blacklist"
     override val description: String = "Hindert einen User daran Commands auszuführen."
-    override val usage: String = "<playerid>"
     override val permission: Permission = Permission.ADMIN
     override val category: CommandCategory = CommandCategory.MODERATION
     override val commandPlace: CommandPlace = CommandPlace.ALL
 
     init {
         registerCommands(BlacklistListCommand())
+        registerCommands(BlacklistToggleCommand())
     }
 
-    override suspend fun execute(context: Context) {
-        val user = context.args.optionalUser(0, jda = context.jda)
-            ?: return context.sendHelp().queue()
+    private inner class BlacklistToggleCommand : AbstractSubCommand.Command(this) {
+        override val name: String = "toggle"
+        override val description: String = "Fügt hinzu/Entfernt einen Nutzer von der blacklist"
 
-        val blacklisted = transaction {
-            val dcUser = DatabaseDevCordUser.findOrCreateById(user.idLong)
-
-            dcUser.blacklisted = !dcUser.blacklisted
-            dcUser.blacklisted
+        override val options: List<CommandUpdateAction.OptionData> = buildOptions {
+            user("target", "Der Nutzer der zur/von der Schwarzen Liste hinzugefügt/entfernt werden soll") {
+                isRequired = true
+            }
         }
 
-        context.respond(Embeds.success(if (blacklisted) "User zur Blacklist hinzugefügt." else "User aus der Blacklist entfernt."))
-            .queue()
+        override suspend fun execute(context: Context) {
+            val user = context.args.user("target")
+
+            val blacklisted = transaction {
+                val dcUser = DatabaseDevCordUser.findOrCreateById(user.idLong)
```

Comment:
"devcordUser" would probably be a better choice, we don't have such abbreviations in other places either

Reply:
that's where kobolds go background shines